### PR TITLE
[master < T0799-dx] Create Memgraph Cloud FAQ document

### PR DESCRIPTION
**Pull request by kgolubic**
### Description

Created Memgraph Cloud FAQ document

### Pull request type

Please delete options that are not relevant and check the ones that are.
- [x] New documentation page

### Checklist:

- [x] I checked all content with Grammarly
- [x] I performed a self-review of my code
- [x] I made corresponding changes to the rest of the documentation
- [x] The build passes locally
- [x] My changes generate no new warnings or errors

### CODE REVIEW

File: help-center/faq/cloud-faq.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+## Projects
+
+### How many projects and snapshots can I create? Why I can’t create more than 3 projects? 
+That is the initial limit for new users. If want to create more projects, [let us know](/help-center).
+
+### Why I can’t create more than 5 snapshots? 
+That is the initial limit for new users. If want to create more snapshots,  [let us know](/help-center).
+
+### How do I upgrade a RAM instance?
```
Comment (vpavicic) [inline help-center/faq/cloud-faq.md:30]: https://memgraph.com/docs/memgraph-cloud/cloud-projects#resize-a-project

I would link to this process...

File: help-center/faq/cloud-faq.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+- 4GB RAM Memgraph Cloud project has about 3.40 GB free RAM
+- 8GB RAM Memgraph Cloud project has about 6.7 GB free RAM
+- 16GB RAM Memgraph Cloud project has about 14 GB free RAM
+- 32GB RAM Memgraph Cloud project has about 28 GB free RAM
+
+### I’ve created a new instance and when I try to connect to it: It says unable to connect.
+In general, Memgraph loads all the MAGE algorithms and sometimes it takes few seconds more at the initial state to load them all.
+
+### I’ve paused my project and resumed it, but my Memgraph’s IP is different now? 
+When you pause your project, usually the IP stays the same, but sometimes your IP can be released and a new one will be allocated. You will always have a new IP in the connection details.
```
Comment (vpavicic) [inline help-center/faq/cloud-faq.md:47]: You can always check the IP in the connection details. ?

File: help-center/faq/cloud-faq.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+### I created a 2GB RAM, but Memgraph Labs says there is total of 1.54GB available. Why is that so?
+A par of RAM is allocated to the operating system and other services. They usually take 13-15% of the total RAM. Approximate free RAM is: 
+- 1GB RAM Memgraph Cloud project has about 860 MB free RAM
+- 2GB RAM Memgraph Cloud project has about 1.60 GB free RAM
+- 4GB RAM Memgraph Cloud project has about 3.40 GB free RAM
+- 8GB RAM Memgraph Cloud project has about 6.7 GB free RAM
+- 16GB RAM Memgraph Cloud project has about 14 GB free RAM
+- 32GB RAM Memgraph Cloud project has about 28 GB free RAM
+
+### I’ve created a new instance and when I try to connect to it: It says unable to connect.
```
Comment (vpavicic) [inline help-center/faq/cloud-faq.md:43]: I've created a new instance, and when I try to connect to it, I get an error: Unable to connect.

File: help-center/faq/cloud-faq.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+In general, Memgraph loads all the MAGE algorithms and sometimes it takes few seconds more at the initial state to load them all.
```
Comment (vpavicic) [inline help-center/faq/cloud-faq.md:44]: Upon creating a project, Memgraph loads all the MAGE algorithms so it takes some time to load them all. Wait 30 seconds, and then try to connect again.

File: help-center/faq/cloud-faq.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+### How can I connect to my project?
+
+You can connect to an instance running within the Memgraph Cloud project via **Memgraph Lab**, a visual interface, **mgconsole**, command-line interface, or one of many drivers. You can find detailed instructions in [Memgraph Cloud documentation](/docs/memgraph-cloud/cloud-connect).
+
+### How can I backup my project?
+
+A project is backed up by creating a snapshot with Amazon EBS. You cannot create a snapshot if you are using a 14-day free trial version of Memgraph Cloud. You can find detailed instructions in [Memgraph Cloud documentation](/docs/memgraph-cloud/cloud-projects#back-up-a-project).
+
+### How can I resize my project?
```
Comment (vpavicic) [inline help-center/faq/cloud-faq.md:61]: That's the same as the upgrading my project, so no need for this one :)

File: help-center/faq/cloud-faq.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+### What is the pricing?
+
+You can estimate the cost of Memgraph Cloud's service by selecting your cloud
+region and instance size with our [Cost
+Calculator](https://cloud.memgraph.com/pricing), or you can check them out at
+[Project rates](/docs/memgraph-cloud/payment#project-rates))
+
+### How can I use a coupon that I got for Memgraph Cloud?
+First, you need to add a credit card to your account. Then you can apply the
+coupon.
```
Comment (vpavicic) [inline help-center/faq/cloud-faq.md:19]: Change the title: How can I redeem a coupon...
Change the word apply in the body, with the word redeem

link to https://memgraph.com/docs/memgraph-cloud/payment#add-a-payment-method on the words "add a credit card"
link to https://memgraph.com/docs/memgraph-cloud/payment#add-a-payment-method on the words redeem the coupon

File: help-center/faq/cloud-faq.md [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+
+### What will happen to my instance after the free trial?
```
Comment (vpavicic) [inline help-center/faq/cloud-faq.md:21]: instance or project?

File: help-center/faq/cloud-faq.md [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+Instance will be stopped for next 7 days. If you wish to continue, add a payment
+method.
+
+## Projects
+
+### How many projects and snapshots can I create? Why I can’t create more than 3 projects?
```
Comment (vpavicic) [inline help-center/faq/cloud-faq.md:27]: I would either just have "How many projects and snapshots can I create? " and then give the numbers for both

Or have 2 questions: Why can’t I create more than 3 projects?  Why can’t I create more than 5 snapshots? 

Or: Why can't I create more than 3 projects and 5 snapshots?

File: help-center/faq/cloud-faq.md [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+
+### How many projects and snapshots can I create? Why I can’t create more than 3 projects? 
+That is the initial limit for new users. If you want to create more projects,
+[let us know](/help-center).
+
+### Why I can’t create more than 5 snapshots? 
+That is the initial limit for new users. If you want to create more snapshots,
+[let us know](/help-center).
+
+### Is it possible to upgrade a RAM instance?
```
Comment (vpavicic) [inline help-center/faq/cloud-faq.md:35]: Is it possible to upgrade a project to use more RAM?

File: help-center/faq/cloud-faq.md [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+A par of RAM is allocated to the operating system and other services. They
+usually take 13-15% of the total RAM. Approximate free RAM is: 
+- 1GB RAM Memgraph Cloud project has about 860 MB free RAM
+- 2GB RAM Memgraph Cloud project has about 1.60 GB free RAM
+- 4GB RAM Memgraph Cloud project has about 3.40 GB free RAM
+- 8GB RAM Memgraph Cloud project has about 6.7 GB free RAM
+- 16GB RAM Memgraph Cloud project has about 14 GB free RAM
+- 32GB RAM Memgraph Cloud project has about 28 GB free RAM
+
+### I've created a new instance, and when I try to connect to it, I get an error: Unable to connect.
```
Comment (vpavicic) [inline help-center/faq/cloud-faq.md:49]: I've created a new project, and when I try to connect to the instance, I get an error: Unable to connect.

File: help-center/faq/cloud-faq.md [hunk 11]
```diff
@@ -0,0 +1,83 @@
...
+
+### Why I can’t create more than 5 snapshots? 
+That is the initial limit for new users. If you want to create more snapshots,
+[let us know](/help-center).
+
+### Is it possible to upgrade a RAM instance?  
+Yes, it is. You can find detailed instructions in [Memgraph Cloud
+documentation](/docs/memgraph-cloud/cloud-projects#resize-a-project).
+
+### I created a 2GB RAM, but Memgraph Labs says there is a total of 1.54GB available. Why is that so?
```
Comment (vpavicic) [inline help-center/faq/cloud-faq.md:39]: I've created a project with 2GB RAM, but Memgraph Labs says there is only1.54GB available. Why is that so?

File: help-center/faq/cloud-faq.md (hunk 7 above)
Comment (kgolubic) [reply help-center/faq/cloud-faq.md:21]: Instance. You get a trial for instance, not a project.

File: help-center/faq/cloud-faq.md [hunk 12]
```diff
@@ -14,29 +14,25 @@ region and instance size with our [Cost
...
-That is the initial limit for new users. If you want to create more snapshots,
-[let us know](/help-center).
-
-### Is it possible to upgrade a RAM instance?  
+### Is it possible to upgrade a project to use more RAM?  
 Yes, it is. You can find detailed instructions in [Memgraph Cloud
 documentation](/docs/memgraph-cloud/cloud-projects#resize-a-project).
 
-### I created a 2GB RAM, but Memgraph Labs says there is a total of 1.54GB available. Why is that so?
+### I've created a project with 2GB RAM, but Memgraph Labs says there is only1.54GB available. Why is that so?
```
Comment (vpavicic) [inline help-center/faq/cloud-faq.md:35]: space missing  "only1.54GB"

**Review by vpavicic — APPROVED** [review]

**Pull request merged**